.github/PULL_REQUEST_TEMPLATE/feature_add.md

```diff
@@ -0,0 +1,26 @@
+[contributing]: https://github.com/Together-Java/TJ-Bot/wiki/Contributing
+[code_guidelines]: https://github.com/Together-Java/TJ-Bot/wiki/Code-Guidelines
+[new_issue]: https://github.com/Together-Java/TJ-Bot/issues/new/choose
+
+## Pull-request
+
+- [ ] I have read the [contributing guidelines][contributing].
+- [ ] I have read the [code guidelines][contributing].
+- [ ] I have created a relating [issue][new_issue].
+
+### Changes
+
+- [ ] Internal code
```

Comment:
Maybe:
```suggestion
- [ ] Existing code
```

Reply:
I didn't mean that but it should be that.